dockers/docker-orchagent/docker-init.sh

```diff
@@ -24,4 +24,21 @@ if [ "$VLAN" != "" ]; then
     cp /usr/share/sonic/templates/ndppd.conf /etc/supervisor/conf.d/
 fi
 
+USE_PCI_ID_IN_CHASSIS_STATE_DB=/usr/share/sonic/platform/use_pci_id_chassis
```

Comment:
The asic_id in the DEVICE_METADATA is updated here on load_minigraph 
https://github.com/Azure/sonic-buildimage/blob/07de6a39c083b2cbce19cfddffd32df4cb95f923/src/sonic-config-engine/sonic-cfggen#L389-L395
Can we add the `USE_PCI_ID_IN_CHASSIS_STATE_DB` check here

Reply:
Are you suggesting that we move all the logic from this PR in `sonic-cfggen`? I think this would basically block `config-setup` until chassisd publishes the PCI address of all ASICs to CHASSIS_STATE_DB.